superbench/benchmarks/micro_benchmarks/__init__.py

```diff
@@ -7,7 +7,9 @@
 from superbench.benchmarks.micro_benchmarks.sharding_matmul import ShardingMatmul
 from superbench.benchmarks.micro_benchmarks.computation_communication_overlap import ComputationCommunicationOverlap
 from superbench.benchmarks.micro_benchmarks.kernel_launch_overhead import KernelLaunch
+from superbench.benchmarks.micro_benchmarks.flops_performance import FlopsPerformanceCuda
 
 __all__ = [
-    'MicroBenchmark', 'MicroBenchmarkWithInvoke', 'ShardingMatmul', 'ComputationCommunicationOverlap', 'KernelLaunch'
+    'MicroBenchmark', 'MicroBenchmarkWithInvoke', 'ShardingMatmul', 'ComputationCommunicationOverlap', 'KernelLaunch',
+    'FlopsPerformanceCuda'
```

Comment:
The name is strange becasue the KernelLaunch may also change to KernelLaunchCuda?

Reply:
rename to "GemmFlops"?

Reply:
KernelLaunch needn't change to 'KernelLaunchCuda', since the CUDA and ROCm version of KernelLaunch will have the same CMakeFile.txt and both of them have the same binary name.
But for GLOPs testing for CUDA and ROCm, we should use different tool which will have different binary and different commands.

Reply:
"GemmFlops"

Reply:
Done.